src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -179,6 +184,17 @@ int sd_dhcp_lease_get_captive_portal(sd_dhcp_lease *lease, const char **ret) {
         return 0;
 }
 
+int sd_dhcp_lease_get_ipv6_only_preferred(sd_dhcp_lease *lease, uint32_t *ret) {
```

Comment:
I'd like to explicitly name the return value is in seconds, `_only_preferred()` -> `_only_preferred_sec()`.